.github/workflows/checks.yml

```
@@ -21,7 +21,7 @@ jobs:
 
       - name: Remove files irrelevant files that may cause TypeScript errors
         run:
-          rm -f ./seed.mts
+          rm -f supabase/seed/seed.mts
```

just to be consisent
```suggestion
          rm -f ./supabase/seed/seed.mts
```